crates/control-interface/src/types/host.rs

```diff
@@ -59,6 +59,7 @@ pub struct HostInventory {
     pub friendly_name: String,
     /// The host's labels
     #[serde(default)]
+    #[serde(serialize_with = "serialize_as_btreemap")]
```

Comment:
Dumb question, do we need to have a deserialize_with here too? Looks like the test works at least 😄

Reply:
So to avoid a breaking change here I did not actually change the type -- it doesn't make sens to try and re-serialize as the `BTreeMap` because the `HashMap` it's turning into is unordered, so we can't count on that anyway.